distros/spark/homescreens.json

```diff
@@ -0,0 +1,42 @@
+{
+  "homescreens": [
+    [
+      ["apps", "communications", "dialer"],
+      ["apps", "sms"],
+      ["apps", "communications", "contacts"],
+      ["apps", "email"],
+      ["apps", "search"],
+      ["apps", "gallery"],
+      ["apps", "camera"],
+      ["apps", "marketplace.firefox.com"],
+      ["apps", "clock"],
+      ["apps", "settings"]
+    ], [
+      ["outoftree_apps", "foxfooding"],
```

Comment:
The Spark apps should go here. These include:
* customizer-launcher
* directory
* sharing
* studio